content/08-cardano-sidechains/03-example-evm-sidechain/06-transferring-tokens.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Transferring tokens
+metaTitle: Transferring tokens
+---
+
+This tutorial will show you how to transfer `SC_Tokens` between the main chain and a sidechain.
+
+## Prerequisites
+
+1. We will use the **sidechain-cli** and the **sc-evm-cli** components within this tutorial. Download the binaries and add them to your shell’s path: https://github.com/input-output-hk/sidechains-tooling.
```

Comment:
Perhaps point to the binaries instead of documentation?
https://github.com/input-output-hk/sidechains-tooling/releases